adhocteam/Head-Start-TTADP
# Ttahub 174/update list of topics

## Description of change
- Update topic names in the front end javascript constants in which they live
- Write a recurring job to execute a sequelize update function to rename the offending topics
- Added unit test for job

## How to test
Create a new activity report, go to the topics and resources pane

Confirm that the topics are in alphabetical order and that:
- "CLASS: Classroom Organization" 
- "Behavioral/Mental Health / Trauma"
- "Curriculum (Instructional or Parenting)"
- "Environmental Health and Safety / EPRR"
- "Quality Improvement Plan/QIP"

appear.

To test recurring job aside from unit test, you'll need to run the updateTopicNames function and have some of the un-renamed topics in the database. As an example, to run it locally for me, I added it to an API handler so it would run arbitrarily, and ran SQL statements to confirm that the topics had been renamed.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-174

## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
